### ISSUE TITLE
Accessibility validation is incorrect for Decorative Images

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#176]
## Expected Behavior

Decorative Image must not have text alternative
## Current Behavior

- When selecting **decorative** option the text field for _Short Description_ is hidden (correct), but accessibility validation keep validating _Short Description_ field (wrong).
- When editing the accessibility settings. the _short description_ field is visible (wrong) for decorative images 

## Steps to Reproduce

1. Configure accessibility of an image 
1. Check **decorative** option
    1. Notice that accessibility status shows as 0/1 (should be 1/1)
1. Save and close
1. The image is presented  with "no accessibility" feedback
1. Select to configure accessibility for the same image
1. Accessibility form is loaded with 
   1. **decorative** option checked 
   1. _Short Description_ field visible (wrong)
   1. Accessibility progress cycle invisible  (wrong)

![image](https://user-images.githubusercontent.com/993369/90967268-1ae95680-e4b3-11ea-905a-b3a1b0b40ea1.png)

![image](https://user-images.githubusercontent.com/993369/90967270-250b5500-e4b3-11ea-8e6e-b3389680e684.png)

![image](https://user-images.githubusercontent.com/993369/90967267-12911b80-e4b3-11ea-97f6-64437ff7afd9.png)

## Environment

https://vm2161.kaj.pouta.csc.fi/

## Related issues
 

This interfere in:
Accessibility Report feature #45 
- Will present wrong data
- Could prevent teacher from publish course: if teacher selects a inclusion goal that depends on Images accessibility and validations is wrongly set as misconfigured, thus Inclusion Goal will be report as not accomplished and Publish button will not be enabled (#45)
- If course's inclusion goal does not rely on Image's accessibility, the course will be eventually published, but students will get wrong data about accessibility
